functions/examples/print-top-5s.ts

```diff
@@ -14,11 +14,13 @@ async function printResults() {
   const config = await Firestore.getConfig();
   const strava = new Strava(config.stravaClientId, config.stravaClientSecret);
   const athletes = await Firestore.getRegisteredAthletes();
-  let athletesWithActivities = await strava.getAllAthletesActivities(
+  const athletesWithActivities = await strava.getAllAthletesActivities(
     athletes,
     getPreviousWeekUnix(),
     getCurrentWeekUnix()
   );
+  // const athletesWithActivities = await strava.getAllAthletesActivities(athletes, getCurrentWeekUnix(), now());
```

Comment:
Do we want this?

Reply:
I think so, we don't have another example for current week results, so I and up changing that all the time